src/server/index.js

```diff
@@ -101,6 +101,20 @@ app.prepare()
         logger.info("configuring keycloak");
         server.use(keycloakClient.middleware());
 
+        server.get("/", keycloakClient.checkSso(), async (req, res, next) => {
+            try {
+                const userProfile = authn.getUserProfile(req);
+
+                if (userProfile) {
+                    return res.redirect(`/${NavigationConstants.DASHBOARD}`);
```

Comment:
This works fine as-is, but users don't normally navigate to the `/dashboard` page anywhere else within the DE, since our next config has it aliased to `/`.

I noticed if we redirect here to `/login` instead of `/dashboard`, then this should log the user into the DE as well, and the handler below will redirect the user back to `/`.

Reply:
Actually, never mind this 😊 

I just tested this out in dev mode, and Firefox complained about a redirect loop, so it seems like redirecting to `/dashboard` is our best option.

:shipit: 👍